config/ips.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+const (
+	// DeepSecuritySaaS is
```

Comment:
__[Golint]__ comment on exported const DeepSecurity should be of the form "DeepSecurity ..." ([link](https://sider.review/gh/repos/54808920/pulls/852#issue-4278537))

<sub>posted by Sider</sub>